samples/BikeSharingApp/README.md

```diff
@@ -1,4 +1,100 @@
-# Bike Sharing Sample: Iteratively Develop and Debug Microservices in Kubernetes
+# Deploy the Bike Sharing sample application to Azure Kubernetes Service
 
-## Quickstart
-[Quickstart: Team development on Kubernetes using Azure Dev Spaces](https://docs.microsoft.com/en-us/azure/dev-spaces/quickstart-team-development).
\ No newline at end of file
+BikeSharing is a microservices-based sample application that helps showcase the sandboxing capbilities of [Azure Dev Spaces](https://aka.ms/devspaces). 
+
+Follow the steps below to deploy this sample app to Azure Kubernete Service (AKS).
+
+## Prerequisites
+* [Azure subscription](https://azure.microsoft.com/free)
+* [Azure CLI](https://docs.microsoft.com/cli/azure/install-azure-cli?view=azure-cli-latest)
+* [Helm 2.13 (or greater)](https://github.com/helm/helm/blob/master/docs/install.md)
+
+## Configure Azure resources
+
+1. **Create an Azure Kubernetes Service cluster.** You must create an AKS cluster in a [supported region](https://docs.microsoft.com/azure/dev-spaces/about#supported-regions-and-configurations). The below commands create a resource group called `bikesharing-group` and an AKS cluster called `bikesharing-cluster` in the `eastus` region.
+
+    ```bash
+    GROUP_NAME=bikesharing-group
+    AKS_NAME=bikesharing-cluster
+    LOCATION=eastus
+
+    az group create --name $GROUP_NAME --location $LOCATION
+    az aks create -g $GROUP_NAME -n $AKS_NAME --location $LOCATION --disable-rbac --generate-ssh-keys
+    ```
+
+1. **Enable Azure Dev Spaces on the AKS cluster.**
+    ```bash
+    az aks use-dev-spaces -g $GROUP_NAME -n $AKS_NAME --space master --yes
+    ```
+
+<!-- 1. **Create an Azure Container Registry.** Save the loginServer value from the output because it is used in a later step.
+    ```bash
+    ACR_NAME=bikesharing-container-registry
+    az acr create -g $GROUP_NAME --name $ACR_NAME --sku Basic
+    ```
+
+1. **Set up role access for the cluster and container registry.** Note: The user running these commands needs to be an *owner* of the target resources.
+    ```bash
+    # Save the output from this command, you'll need the `clientId` value for the next commands
+    az ad sp create-for-rbac --sdk-auth --skip-assignment 
+
+    # Get the AKS resource id
+    az aks show -g $GROUP_NAME -n $AKS_NAME --query id -o tsv
+
+    # Create role assignment for access to AKS cluster
+    az role assignment create --assignee <clientId> --scope "<aksResourceId>" --role Contributor
+
+    # Get the container registry (ACR) id
+    az acr show -g $GROUP_NAME -n $ACR_NAME --query id -o tsv
+    
+    # Create role assignment for 'push access' to container registry
+    az role assignment create --assignee  <clientId>  --scope "<acrResourceId>" --role AcrPush
+    ```
+ -->
```

Comment:
Do we need all this configuration given that we created the cluster with --disable-rbac?

Reply:
Hmm alright, I think this is needed for the Connect scenarios probably. :) Closing the comment.

---
In reply to: [339270558](https://github.com/Azure/dev-spaces/pull/185#discussion_r339270558) [](ancestors = 339270558)

Reply:
This whole section is in comments (I didn't even see it when I reviewed by looking at the file).  What's the intent here?

Reply:
This block is currently commented out and not required.

Reply:
Removed block